apps/minifront/src/state/staking/index.test.ts

```diff
@@ -1,5 +1,5 @@
 import { beforeEach, describe, expect, it, vi } from 'vitest';
-import { StoreApi, UseBoundStore, create } from 'zustand';
+import { create, StoreApi, UseBoundStore } from 'zustand';
```

Comment:
it seems like a good chunk of these changes are due to import sorting, can you turn that off

Reply:
lol definitely not